FESS-GUI/mainwindow.cpp

```diff
@@ -100,8 +103,7 @@ MainWindow::MainWindow(QWidget *parent) :
 
 void MainWindow::realtimeDataSlot()  //Important function. This is repeatedly called
 {
-    flywheelOperation->sync();
-    //at the refresh rate defined by the timer (line 107)
+    //at the refresh rate defined by the timer
```

Comment:
Sync is now bound to a signal from a timer so that it can run independently of the graphs.